### ISSUE TITLE
Support alternative source location

### ISSUE THREAD

**Issue by richardmward** [capacitor-community/electron#119]
**Is your feature request related to a problem? Please describe.**
Android and iOS capacitor applications can choose an alternative location for their source using the `android.path` and `ios.path` configuration properties in `capacitor.config.ts`.

**Describe the solution you'd like**
Be able to configure an alternative location for the electron source.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by IT-MikeS** [comment]
I will try and make this part of the v4 release 👍

**Comment by muuvmuuv** [comment]
Yes this would be great! We have a monorepo structure and would love to have the desktop app stuff in `projects/desktop` but running `pnpx cap open ..` returns an error "Dir electron not found" in the root.

**Comment by IT-MikeS** [comment]
Never considered the monorepo use case, I'm hopeful I'll have this in the next release, it's taking longer than I hoped due to life stuff but I'm working on it 👍
